examples/language/lib/patterns/switch.dart

```diff
@@ -45,7 +57,28 @@ void main() {
         print('a = $a, b = $b');
 
       default:
-      // #enddocregion switch-statement
     }
+    // #enddocregion switch-statement
+  }
+
+  {
+    final color = Color.red;
+    // #docregion or-share-body
+    var isPrimary = switch (color) {
+      Color.red || Color.yellow || Color.blue => true,
+      _ => false
+    };
+    // #enddocregion or-share-body
+    isPrimary;
```

Comment:
This isn't important, it just feels weird to have a lone variable there to me.

```suggestion
    print(isPrimary);
```